openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -60,6 +102,23 @@ def to_representation(self, instance):
     def get_tags_count(self, instance):
         return instance.tag_set.count()
 
+    def get_can_tag_object(self, instance) -> Optional[bool]:
+        """
+        Returns True if the current request user may create object tags on this taxonomy.
+
+        (The object_id test is necessarily skipped because we don't have an object_id to check.)
+        """
+        request = self._request
+        assert request and request.user
+        if not self._include_perms:
+            return None
+
+        model = self._model
+        app_label = model._meta.app_label
+        perm_name = f'{app_label}.add_objecttag'
+        perm_object = ObjectTagPermissionItem(taxonomy=instance, object_id="")
+        return request.user.has_perm(perm_name, perm_object)  # type: ignore[arg-type]
+
```

Comment:
@rpenido 

Since people may override these rules when they use the library, I'd rather call the canonical permission, and let the rule itself [call `change_objecttag_taxonomy`](https://github.com/openedx/openedx-learning/blob/main/openedx_tagging/core/tagging/rules.py#L152-L155).

Reply:
Sure! Either way is acceptable to me.